creme/optim/losses.py

```diff
@@ -106,5 +130,4 @@ def __call__(self, y_true, y_pred):
     def gradient(self, y_true, y_pred):
         if y_pred * y_true < 1:
             return - y_pred * y_true
```

Comment:
I'm really not sure but isn't the gradient of the Hinge loss `-y`? I just looked at [https://stats.stackexchange.com/questions/4608/gradient-of-hinge-loss](https://stats.stackexchange.com/questions/4608/gradient-of-hinge-loss).